_summaries/ulyanov2016instance_1.md

```diff
@@ -1,19 +1,64 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: ulyanov2016instance
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: rll2396
+score: 4
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+* Core Idea
+
+    Instance normalization can help a feed-forward generator network achieve better performance at stylizing images.
```

Comment:
I think you can also a brief description of what the problem Instance normalization is solving.